src/fruit/migrations/0009_copy-position.py

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.contrib.gis.geos import Point
+from django.db import migrations
+
+
+def copy_position(apps, schema_editor):
+    Fruit = apps.get_model("fruit", "Fruit")
+
+    for fruit in Fruit.objects.all():
+        fruit.position = Point(float(fruit.longitude), float(fruit.latitude))
```

Comment:
Tohle trvá dlouho a dalo by se to asi udělat přes update() a F() object. Ale vem to čert, máme rychlé stroje:)